sensor/kubernetes/sensor/sensor.go

```diff
@@ -155,10 +154,8 @@ func CreateSensor(cfg *CreateOptions) (*sensor.Sensor, error) {
 	// i.e., after nodeInventoryHandler
 	components = append(components, nodeInventoryHandler, complianceMultiplexer)
 
-	if features.ComplianceEnhancements.Enabled() {
-		coInfoUpdater := complianceoperator.NewInfoUpdater(cfg.k8sClient.Kubernetes(), 0)
-		components = append(components, coInfoUpdater, complianceoperator.NewRequestHandler(cfg.k8sClient.Dynamic(), coInfoUpdater))
-	}
+	coInfoUpdater := complianceoperator.NewInfoUpdater(cfg.k8sClient.Kubernetes(), 0)
+	components = append(components, coInfoUpdater, complianceoperator.NewRequestHandler(cfg.k8sClient.Dynamic(), coInfoUpdater))
```

Comment:
Not something that was changed by you, but why is `coInfoUpdater` being added twice to the slice of components?

Reply:
@fredrb I don't think it is.  
`coInfoUpdater` is added as a component.  Then `coInfoUpdater` is passed to the request handler as it has helper functions that are used by the request handler.

`coInfoUpdater` watches compliance CRs whereas the request handler services messages from central.  So I think we are adding 2 distinct things to the components list.

Reply:
My bad, judged too fast